HW03/andrei_godes/andrei_godes_hw_03_t_03.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'time'
+class Homework3
+  def task3(logs)
+    array = subtraction(include_string(logs))
```

Comment:
Could you please give more descriptive variable names?
The 'array' doesn't say anything about what's in there (it applies to everything else too)
Try to give descriptive names to variables that will reflect what you put in them